packages/shared/src/components/widgets/ReferralWidget.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import { Button, ButtonSize } from '../buttons/Button';
+import { TextField } from '../fields/TextField';
+import { link } from '../../lib/links';
+import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
+import { labels } from '../../lib';
+import { AnalyticsEvent, TargetId } from '../../lib/analytics';
+import ReferralSocialShareButtons from './ReferralSocialShareButtons';
+
+const ReferralWidget = ({ url }: { url: string }): ReactElement => {
+  const inviteLink = url || link.referral.defaultUrl;
+  const [, onShareOrCopyLink] = useShareOrCopyLink({
+    text: labels.referral.generic.inviteText,
+    link: inviteLink,
+    trackObject: () => ({
+      event_name: AnalyticsEvent.CopyReferralLink,
+      target_id: TargetId.ProfilePage,
+    }),
+  });
+  const onShareClick = () => {
+    onShareOrCopyLink();
+  };
+
+  return (
+    <div
+      data-testid="referral-widget"
+      className="flex laptopL:absolute flex-col tablet:flex-row laptopL:flex-col flex-wrap justify-between p-4 laptopL:m-6 mt-6 laptopL:mt-0 mb-4 laptopL:max-w-widget laptopL:h-auto rounded-2xl border border-theme-divider-tertiary laptopL:left-[40rem] max-w-fit bg-theme-bg-primary"
+    >
+      <h3 className="mb-2 font-bold typo-title3">Invite friends</h3>
```

Comment:
From the Sketch I think it says title 2.
```suggestion
      <h3 className="mb-2 font-bold typo-title2">Invite friends</h3>
```

Reply:
Yes but the size in sketch is 20px which is title 3, from design it also looks like it makes sense to be title3 as it's the same as the user name in the profile... 🤔

Reply:
I see it's set as typo-2 but design overwritten.
The general rule of thumb would be to take whatever the name is set to, we don't really look at pixel values if that makes sense?

But when we do see these overwrites we should notify design it just means something went wrong in the sync most of the time. (See little warning signs)

![Screenshot 2023-11-14 at 08 42 18](https://github.com/dailydotdev/apps/assets/554874/835c91e3-f346-4e6d-8d50-be091e39ce74)

Reply:
True true, I've missed this in my list of questions for design

Reply:
@rebelchris thank you for clearing it up! Seems like they wanted title3 anyway. So we should be good here 🚀